draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -627,10 +627,34 @@ in cache before performing any followup queries.
 With these optimizations in place, and conforming DNS servers,
 using SVCB does not add network latency to connection setup.
 
+Moreover, if an A or AAAA response arrives before an SVCB response, the
+client SHOULD wait up to CD milliseconds before connecting as if the
```

Comment:
The normative force here is unclear.  Is optimistic pre-connection RECOMMENDED?  What does it mean to wait for "up to CD milliseconds"?

I would prefer for optimistic pre-connection to be OPTIONAL.

Reply:
SHOULD and RECOMMENDED are the same per RFC2119, right? I'm fine bumping it down to optional advice, since it truly is optional.

> What does it mean to wait for "up to CD milliseconds"?

Is "wait up to CD milliseconds before connecting" not clear? It means, "don't try and connect until CD ms go by." How can we make this more clear?

Reply:
"SHOULD wait ... before connecting" is ambiguous.  Is it "SHOULD wait and SHOULD connect" or "MAY connect but SHOULD wait"?  I would prefer the latter.  If indirection (non-empty SvcDomainName) turns out to be the most common reason why SVCB is slow, the success rate for this trick might be too low to justify the complexity.

It sounds like "wait *at least* CD milliseconds" would better capture your meaning.

Reply:
> It sounds like "wait at least CD milliseconds" would better capture your meaning.
 
Indeed it would! I'll use that.